### PR TITLE
[Zone] Ashenvale

### PR DESCRIPTION
**Pull request by Knindzagxg**
Ashenvale is now fully updated, since every creature in zone was missing SmartAI.

Working on every zone at the moment, and will post updates once i'm done with each.

To test it:
Apply SQL
Restart your server

You can manually spawn NPC by ID to check SmartAI or you can go to zone and check like that as well.

.npc add ID
or
.tele Ashenvale and run / test around.

### CODE REVIEW

**Comment by merdlin** [comment]
Tested, it works.

**Comment by FrancescoBorzi** [comment]
@merdlin thanks, did you test all creatures from the PR?

File: data/sql/updates/pending_db_world/rev_1589718566727079200.sql [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+(3814, 0, 0, 0, 0, 0, 100, 0, 2400, 3100, 8600, 9900, 11, 3391, 0, 0, 0, 0, 0, 2, 0, 0, 0, 0, 0, 0, 0, 'Spined Crawler - In Combat - Cast \'3391\'');
+
+UPDATE `creature_template` SET `AIName` = 'SmartAI' WHERE `entry` = 3809;
+DELETE FROM `smart_scripts` WHERE (source_type = 0 AND entryorguid = 3809);
+INSERT INTO `smart_scripts` (`entryorguid`, `source_type`, `id`, `link`, `event_type`, `event_phase_mask`, `event_chance`, `event_flags`, `event_param1`, `event_param2`, `event_param3`, `event_param4`, `action_type`, `action_param1`, `action_param2`, `action_param3`, `action_param4`, `action_param5`, `action_param6`, `target_type`, `target_param1`, `target_param2`, `target_param3`, `target_x`, `target_y`, `target_z`, `target_o`, `comment`) VALUES
+(3809, 0, 0, 0, 0, 0, 100, 0, 2100, 2400, 9400, 10800, 11, 15793, 0, 0, 0, 0, 0, 2, 0, 0, 0, 0, 0, 0, 0, 'Ashenvale Bear - In Combat - Cast \'15793\'');
```
Comment (Kitzunu) [inline data/sql/updates/pending_db_world/rev_1589718566727079200.sql:76]: Newline pls

**Comment by merdlin** [comment]
> @merdlin obrigado, você testou todas as criaturas do PR?

Yes

**Pull request merged**